apps/signup/signup.html

```diff
@@ -109,7 +109,7 @@ <h2 style="margin-top: -.5em;">User Signup</h2>
           If you are an Admin, you can directly signup users.
         </p>
       </form>
-      <div class="text-center" onclick="loginPage()">Already have an account? </div>
+      <div class="text-center"><a href="/login.html">Already have an account?</a> </div>
```

Comment:
I'd prefer to use relative paths whenever possible, as we often see deployments at urls like http://example.com/pathology/caMicroscope/

```suggestion
      <div class="text-center"><a href="../..//login.html">Already have an account?</a> </div>
```

Reply:
@birm I have made the requested change to add `../..//login.html`. Could you please review it and confirm if it aligns with your project's standards?